docs/openapi/api-external-interop-v1.yaml

```diff
@@ -1,327 +1,394 @@
 openapi: 3.0.3
 info:
-  title: 'E-Service Monitor Interop API specifications'
+  title: E-Service Monitor Interop API specifications
   version: 1.0.0
-  x-summary: 'E-Service Monitor APIs'
-  description: >- 
-    OpenAPI specification for E-Service Monitor's services exposed for the Interop platform
+  x-summary: E-Service Monitor APIs
+  description: OpenAPI specification for E-Service Monitor's services exposed for the Interop platform
   termsOfService: https://www.e-serviceurl-example.com/terms
   contact:
     name: contactToDefinePagoPa
     email: toDefinePagoPa@pagopa.it
   license:
     name: Interop's licence
-    url: 'https://to-define/'
+    url: https://to-define/
 servers:
-- url: 'https://to-define/'
+- url: https://to-define/
   description: Development environment
 tags:
-  - name: EServices
-    description: >-
-      Operations on EServices registry
-  - name: Probing
-    description: >-
-      Operations on probing settings
-
+- name: EServices
+  description: Operations on EServices registry
+- name: Probing
+  description: Operations on probing settings
 paths:
-   '/eservices/{eserviceId}/versions/{versionId}/updateState':
+  '/eserviceOperations/{eserviceId}/versions/{versionId}/updateState':
```

Comment:
this has not the same meaning.
the `eservices` prefix is a RESTful standard/convention, while `eserviceOperations` is the prefix for this service